docs/claw/feishu.md

```
@@ -0,0 +1,186 @@
+
+## Verify the Connection
+
+Start the Claw gateway and check the logs:
+
+```bash
+python -m datus.claw.main --config conf/agent.yml --debug
+```
+
+You should see:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)